schema/testdata/good/spec-example.json

```diff
@@ -17,6 +17,7 @@
       "FOO=CDI_SPEC",
       "BAR=BARVALUE1"
     ],
+    "user": {"uid": 1000, "gid": 1000, "umask": 7, "additionalGids": [2, 8]},
```

Comment:
kk and I think we need some additional spec-examples.. or modify the basic test case for handling omitempty variants in test..